Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml

```diff
@@ -15,13 +15,13 @@
     fixtures:
     - shape:
         !type:PhysShapeAabb
-          bounds: "-0.45,-0.45,0.45,0.45"
+        bounds: "-0.45,-0.45,0.45,0.45"
       mass: 100
       mask:
-      - MobMask
+      - FullTileMask
       layer:
-      - MobImpassable
-      - VaultImpassable
+      - HighImpassable
+      - MidImpassable
```

Comment:
IMO (not saying this has to be done) any layers/masks that have more than one entry should probably be coalesced into a single enum with a descriptive name, which you already did for the majority of uses

Reply:
Hah, alright. I didn't want to go overboard with the new CollisionGroups since this one only has the two and it's not similar to any of the others, but I can add it.

Reply:
yehh, maybe this one in particular doesnt but i noted some others that could have some added